python/paddle/fluid/layers/nn.py

```diff
@@ -12811,3 +12812,38 @@ def shard_index(input, index_num, nshards, shard_id, ignore_value=-1):
         },
         stop_gradient=True)
     return out
+
+
+@templatedoc()
+def hard_swish(x, threshold=6.0, scale=6.0, offset=3.0, name=None):
+    """
+    ${comment}
+    Args:
+        x(Varaible): Input of HardSwish operator.
+        threshold(float): The threshold value of Relu6. Default:threshold=6.0
+        scale(float): The scale parameter for the input. Default:scale=6.0
+        offset(float): The offset parameter for the input. Default:offset=3.0
+        name(str|None): A name for this layer(optional). If set None, the layer
+                        will be named automatically.
+
+    Returns:
+        Variable: Output of HardSwish operator.
```

Comment:
HardSwish operator -> hard_swish layer

Reply:
please add "same shape as x" to show that output shape is the same as x